package.json

```diff
@@ -12,7 +12,7 @@
     "html5",
     "appc-client"
   ],
-  "version": "1.17.3",
+  "version": "1.18.0",
```

Comment:
It is usually not a good idea to increase version in PR.
Maintainers will do it on their own.